README.md

```diff
@@ -1,19 +1,61 @@
 # Open Trading Research Infrastructure
-Open Trading Research Infrastructure.
 
-## config.json
+**Open Trading Research Infrastructure (OTRI)** is a research platform for financial computing focusing on data retrieval, validation, and analysis.
+
+## Getting started
+
+### Requirements
+
+- Any OS that supports Python 3.7
+- Python 3.7 and PIP
+
+### Installation
+
+```bash
+$~: git clone https://github.com/OTRI-Unipd/OTRI
+$~: cd OTRI
+$~: pip install -r requirements.txt
+```
+
+### Database
+
+Currently, only a PostgreSQL database is supported.
+The structure of the database is the following:
+
+- An `atoms` table that contains all retrieved data [ID : BigSerial, data_json : jsonb]
```

Comment:
Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5985894)